src/core/Parameters.cc

```diff
@@ -261,3 +278,27 @@ void print_argConvert_warning(
   message << ", found " << Value::typeName(found.type()) << " " << "(" << found.toEchoStringNoThrow() << ")";
   LOG(message_group::Warning, loc, documentRoot, "%1$s", message.str());
 }
+
+void print_argConvert_warning(
+  const std::string& calledName,
+  const std::string& argName,
+  const Value& found,
+  std::vector<Value::Type> expected,
+  const Location& loc,
+  const std::string& documentRoot
+  ) {
+  std::stringstream message;
+  message << calledName << "(..., " << argName << "=" << found.toEchoStringNoThrow() << ") parameter could not be converted: expected ";
```

Comment:
You could drop the "parameter could not be converted" part. You've given the name in context, so it is clear that it is a parameter you are complaining about, and the "expected x, found y" makes the reason for the displeasure clear.

Reply:
I agree the wording there is odd since there isn't really any "conversion" attempted.
I am guessing this phrase originally comes from situations like `cube(size=1);` where size gets "converted" to a "vec3" `[1,1,1]`.

It feels like its missing something to remove the phrase entirely though.  I'm thinking maybe replace with "Invalid type:"